src/Api/Serializer/Ec2ParamBuilder.php

```diff
@@ -12,7 +12,7 @@ class Ec2ParamBuilder extends QueryParamBuilder
     protected function queryName(Shape $shape, $default = null)
     {
         return ($shape['queryName']
-            ?: ucfirst($shape['locationName']))
```

Comment:
I probably gave wrong directions here initially; the default should just be changed to `''` if this is only to fix the warning.  If it's to fix an additional warning around $shape['locationName'] not being set, then it should look more like 
```
        return ($shape['queryName']
            ?: isset ($shape['locationName'])) 
                ? ucfirst($shape['locationName'])
                : "";
```